src/pages/Home/Home.jsx

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import "./home.css";
+import { heroImg } from "assets";
+import { CategoryCard } from "components";
+import { db } from "firebase-config";
+import { Link } from "react-router-dom";
+import { useEffect, useState } from "react";
+import { collection, getDocs } from "firebase/firestore";
+
+export function Home() {
+  const [categories, setCategories] = useState([]);
+  useEffect(() => {
+    const categoriesCollectionRef = collection(db, "categories");
+    (async () => {
+      try {
+        const data = await getDocs(categoriesCollectionRef);
+        const allCategories = data.docs.map((doc) => ({
+          ...doc.data(),
+          id: doc.id,
+        }));
+        setCategories(allCategories);
+      } catch (err) {
+        console.log(err);
```

Comment:
suggestion - you can use `console.error` here

Reply:
sure thanks for suggesting that.